curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/66694fc4bba24f33ca01fa5b.md

```diff
@@ -0,0 +1,85 @@
+---
+id: 66694fc4bba24f33ca01fa5b
+title: Step 3
+challengeType: 20
+dashedName: step-3
+---
+
+# --description--
+
+Once you have a list, most often you can't add items directly to it changing the declaration.
```

Comment:
```suggestion
Once you have a list, you generally won't add items to it by changing the declaration.
```

Not sure what we are trying to say here, tbh.

Reply:
I assume it is trying to explain that you do not add new items by re-declaring/re-initializing the list with new values. Although, it seems like an odd, or incomplete, preamble to teaching list methods.

I might suggest directly going into the methods without the preamble.

> Python has a handful of list methods. Such as methods for adding or removing list items.